backend/prisma/schema.prisma

```diff
@@ -0,0 +1,23 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model User {
+  id        Int    @id @default(autoincrement())
+  email     String @unique
+  username  String @unique
+  firstName String
+  lastName  String
+  password  String @default(dbgenerated("gen_random_uuid()")) // Random password assignment
```

Comment:
As generating a password and saving it in clear is obviously a bad idea. I guess we will change this very soon. You could have just keep this part in the code so you avoid a database migration.

Reply:
You're absolutely right about this, I forgot to remove it. Thanks for the heads up

Reply:
Thank you for the insight, I opened an issue to remind myself: https://github.com/enekesabel/DDDForum/issues/3